package.json

```diff
@@ -42,7 +42,7 @@
   "dependencies": {
     "@diplodoc/latex-extension": "^1.3.2",
     "@diplodoc/mermaid-extension": "^1.3.3",
-    "@diplodoc/transform": "^4.48.1",
+    "@diplodoc/transform": "^4.51.0",
```

Comment:
update this package via action please
https://github.com/diplodoc-platform/client/actions/workflows/deps.yml